tests/tests_geomstats/test_visualization.py

```diff
@@ -85,14 +88,54 @@ def test_coordinates_pre_shape(self):
         expected = .25 * gs.ones(self.n_samples)
         self.assertAllClose(result, expected)
 
-    def test_rotation_pre_shape(self):
+    def test_rotation_pre_shape_2d(self):
         theta = gs.random.rand(1)[0]
         phi = gs.random.rand(1)[0]
         rot = self.KS.rotation(theta, phi)
         result = _SpecialOrthogonalMatrices(3).belongs(rot)
         expected = True
         self.assertAllClose(result, expected)
 
+    def test_draw_pre_shape_3d(self):
+        self.KD.draw()
+
+    def test_plot_points_pre_shape_3d(self):
+        points = self.S33.random_point(self.n_samples)
+        visualization.plot(points, space='S33')
+        points = self.M33.random_point(self.n_samples)
+        visualization.plot(points, space='M33')
+        self.KD.clear_points()
+
+    def test_plot_curve_pre_shape_3d(self):
```

Comment:
NIT: try to use the name of the function tested within the name of the test function, e.g.:
`test_draw_curve_pre_shape_3d`, `test_draw_vector_pre_shape_3d`, `test_convert_to_planar_coordinates_pre_shape_3d`, etc.

The idea is that each test is "unit", i.e. it tests one function only. So, when the tests fail, we know directly which functions are failing just by looking at the names of the tests.